pkg/backup/backuper_test.go

```diff
@@ -0,0 +1,184 @@
+package backup
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBackuperDefault(t *testing.T) {
+	// substitutes the open function to a mocked version
+	open = func(uri string) (DB, error) {
+		db, err := openDatabase(uri)
+		require.NoError(t, err)
+
+		return &tempDatabase{db}, nil
+	}
+
+	// substitutes the newBackupFile function to a mocked version
+	newBackupFile = func(dir string, _ time.Time) (string, time.Time, error) {
```

Comment:
This mocked version is just to fix the timestamp of the backup, so we can test the filename.